coordinator/src/services/translation/request/agent-attended.ts

```diff
@@ -0,0 +1,47 @@
+import {hl7V3} from "@models"
+
+import pino from "pino"
+
+import {createRepresentedOrganization} from "./agent-unattended"
+
+export async function createAuthorForAttendedAccess(
+  professionalCode: string,
+  logger: pino.Logger,
+  organizationCode?: string,
+): Promise<hl7V3.Author> {
+  const agentPerson = await createAgentPersonForAttendedAccess(professionalCode, logger, organizationCode)
+  const author = new hl7V3.Author()
+  author.AgentPerson = agentPerson
```

Comment:
Just thinking a bit, could we add an optional arguement to the `Author` constructor for `AgentPerson` to avoid having to do this line here?